src/eXpansion/Bundle/WidgetCurrentMap/Plugins/Gui/CurrentMapWidgetFactory.php

```diff
@@ -91,11 +91,14 @@ protected function createContent(ManialinkInterface $manialink)
 
         $manialink->getFmlManialink()->getScript()->addCustomScriptLabel(ScriptLabel::OnInit,
             <<<EOL
-            
+            log(Now);
```

Comment:
Log? error?

Reply:
debug, found out that i had to change the dataprovider for more generic as of legacy one.